ui/app/templates/allocations/allocation/index.hbs

```diff
@@ -173,6 +174,7 @@
               <td data-test-service-name>{{row.model.name}}</td>
               <td data-test-service-port>{{row.model.portLabel}}</td>
               <td data-test-service-tags>{{join ", " row.model.tags}}</td>
+              <td data-test-service-onupdate>{{row.model.onUpdate}}</td>
```

Comment:
I was unsure about this data tag and guessed

Reply:
The data tags are arbitrary, so this is fine. There is another step though (see later comment).